src/main/java/christmas/model/Benefit.java

```diff
@@ -0,0 +1,93 @@
+package christmas.model;
+
+import christmas.repository.MenuBoard;
+import java.util.EnumMap;
+
+public class Benefit {
+    private static final int FIRST_DAY = 1;
+    private static final int MINIMUM_UNIT = 1000;
+    private static final int SALE_PRICE = 2023;
+    private static final int PROMOTION_PRICE = 120_000;
+    private static final int MINIMUM_ORDER_PRICE = 10_000;
+    private static final long DAY_DISCOUNT = 100L;
+
+    private final EnumMap<DiscountType, Long> result = new EnumMap<>(DiscountType.class);
+    private final OrderedMenus orderedMenus;
+    private final EventDay eventDay;
+    private long totalDiscountAmount;
+
+    public Benefit(EventDay eventDay, OrderedMenus orderedMenus) {
+        this.eventDay = eventDay;
+        this.orderedMenus = orderedMenus;
+        makeResult();
+    }
+
+    public void makeResult() {
+        if (isOverEventPrice(orderedMenus.totalAmount())) {
+            result.put(DiscountType.CHRISTMAS, christmas(eventDay));
+            result.put(DiscountType.WEEKDAY, weekDay(eventDay, orderedMenus));
+            result.put(DiscountType.WEEKEND, weekend(eventDay, orderedMenus));
+            result.put(DiscountType.SPECIAL, special(eventDay));
+            result.put(DiscountType.PROMOTION, promotion(orderedMenus.totalAmount()));
+        }
+    }
+
+    public long totalBenefitAmount() {
+        return result.values().stream().mapToLong(Long::longValue).sum();
```

Comment:
(추천드려요!)
가독성을 위해 줄바꿈 해주는 것도 좋은 방법입니다 :)
```suggestion
        return return result.values()
                .stream()
                .mapToLong(Long::longValue)
                .sum();
```